src/app.js

```diff
@@ -2,5 +2,9 @@ function app() {
   const invite = 'Hello from app.js';
   console.log(invite);
 }
+export default app
 
-export default app;
+import createBtn from "./btn";
+
+const btn = createBtn
+btn()
```

Comment:
This order is incorrect (eslint says it for sure too), you shouldn't keep any code below `export default`. Only one line break should be there.